README.md

```diff
@@ -1,2 +1,2 @@
-# anomaly-detection
-Repo for anomaly detection
+# orchestration
+Repo for automation and orchestration
```

Comment:
Copy and paste error.  Remove these changes.  The original description is correct.

Reply:
done